gulp/util/handleErrors.js

```diff
@@ -0,0 +1,19 @@
+var gutil = require('gulp-util'),
```

Comment:
I need to update the generator, I found a better way to handle edge-case errors from browserify. Checkout pixi's `handleErrors.js` file for an example:

https://github.com/GoodBoyDigital/pixi.js/blob/dev/gulp/util/handleErrors.js